core/src/main/scala/com/databricks/labs/remorph/parsers/intermediate/functions.scala

```diff
@@ -279,6 +280,7 @@ class CallMapper {
     case CallFunction("TRANSLATE", args) => StringTranslate(args.head, args(1), args(2))
     case CallFunction("TRIM", args) => StringTrim(args.head, args(1))
     case CallFunction("TRUNC", args) => TruncDate(args.head, args(1))
+    case CallFunction("TRY_TO_NUMBER", args) => TryToNumber(args.head, args(1))
```

Comment:
we don't have `TryToNumber ` in Spark. Make the conversion in optimizer rule

Reply:
https://docs.databricks.com/en/sql/language-manual/functions/try_to_number.html